File: 백준/Silver/2563. 색종이/색종이.java
import java.io.*;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        
        int N = Integer.parseInt(br.readLine());
        int width = 0;
        
        boolean[][] arr = new boolean[100][100];
        
        for (int i = 0; i < N; i++) {
            
            StringTokenizer st = new StringTokenizer(br.readLine());
            
            int x = Integer.parseInt(st.nextToken());
            int y = Integer.parseInt(st.nextToken());
            
            for (int j = x; j < x + 10; j ++) {
                for (int k = y; k < y + 10; k++) {
                    if (!arr[j][k]) {
                        arr[j][k] = true;
                        width++;
                    }
                }
            }
        }
        System.out.println(width);
        
        br.close();
    }
}

File: 백준/Silver/2563. 색종이/색종이.kt
import java.io.BufferedReader
import java.io.InputStreamReader

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {
    val N = readLine().toInt()
    val map = Array(101) { Array(101) { false } }
    var result = 0

    repeat(N) {
        val (x, y) = readLine()!!.split(" ").map { it.toInt() }

        for (i in x until x + 10) {
            for (j in y until y + 10) {
                if (!map[i][j]) {
                    map[i][j] = true
                    result++
                }
            }
        }
    }
    println(result)
}